### PR TITLE
Update Curation Concerns

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: spec/forms/curation_concerns/multi_volume_work_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe CurationConcerns::MultiVolumeWorkForm do
+  subject { described_class.new(work, nil) }
+  let(:work) { MultiVolumeWork.new }
+
+  it "doesn't initialize description" do
```
Comment (escowles) [inline spec/forms/curation_concerns/multi_volume_work_spec.rb]: Did we decide that MultiVolumeWorks shouldn't have a `description`?

**Comment by jpstroop** [comment]
They should.

File: spec/forms/curation_concerns/multi_volume_work_spec.rb (hunk 1 above)
Comment (jpstroop) [inline spec/forms/curation_concerns/multi_volume_work_spec.rb]: They should.

Comment (tpendragon) [inline spec/forms/curation_concerns/multi_volume_work_spec.rb]: This shouldn't stop that. It just makes it not fill a new record with a default of an empty array.

**Pull request merged**